src/update_engine_ie9.jsx

```diff
@@ -64,8 +64,10 @@ export function update() {
  */
 export const requestAsyncUpdate = _.debounce(updateAfterDelay, 0);
 
-// update on window resize
-window.addEventListener('resize', _.debounce(requestAsyncUpdate, 16));
+if (typeof window !== 'undefined') {
```

Comment:
do we need this? server side rendering probably won't `require` this file anyway